Makefile

```diff
@@ -32,9 +32,12 @@ $(call VENDOR_RUN,${PROTOC_GEN_GOFAST_GITHUB})
 $(call VENDOR_RUN,${PROTOC_GEN_GO_GITHUB}) protoc -Ivendor -I. --plugin=${BUILD_TOOLS}/protoc-gen-gogofaster --gogofaster_out $$GOPATH/src $1
 endef
 
+export GO111MODULE=off
```

Comment:
why turning of on 1.11 only ?
should be always auto

Reply:
will remove leaving default. in 1.11 its' auto; it will change to on in future see: https://github.com/golang/go/issues/31857